sqlair.go

```diff
@@ -163,3 +164,80 @@ func (q *Query) One(outputArgs ...any) error {
 	iter.Decode(outputArgs...)
 	return iter.Close()
 }
+
+// All iterates over the query and decodes all rows into the provided slices.
+//
+// For example:
+//
+//	var pslice []Person
+//	var aslice []*Address
+//	err := query.All(&pslice, &aslice)
+func (q *Query) All(sliceArgs ...any) (err error) {
```

Comment:
Can we please have the additional `//` between the commented and uncommented code?  It looks significantly more clear for cases like this.

Reply:
`go fmt` removes it!
I'd like to put one there but I cant see a elegant way to do it apart from adding back in a comment below, which I have just done.

Reply:
Ouch, thanks.